Cargo.toml

```diff
@@ -49,6 +49,7 @@ derive = ["zerocopy-derive"]
 alloc = []
 simd = []
 simd-nightly = ["simd"]
+std = []
```

Comment:
This is here because I added a `ParserError` type.  When `std` is available, `ParserError` implements the `std::error::Error` trait.